client/src/legacy/GridField.js

```diff
@@ -41,14 +67,21 @@ $.entwine('ss', function($) {
         ajaxOpts.data = window.location.search.replace(/^\?/, '') + '&' + $.param(ajaxOpts.data);
       }
 
-      form.addClass('loading');
+      // Enable loading animation
+      tbody.find('tr').remove();
+      var loadingCell = $('<td />')
+        .addClass('ss-gridfield-item loading')
+        .attr('colspan', colspan);
+      tbody.append($('<tr />').append(loadingCell));
 
-      $.ajax($.extend({}, {
+      var request = $.ajax($.extend({}, {
         headers: {"X-Pjax" : 'CurrentField'},
         type: "POST",
         url: this.data('url'),
         dataType: 'html',
-        success: function(data) {
+        async:true,
```

Comment:
Space after `:`

Reply:
`async:true` should not have been there ... that was a remnant of me knocking my head against the wall and trying stuff.